reports/ui/etc/nginx/conf.d/default.conf

```diff
@@ -0,0 +1,13 @@
+server {
```

Comment:
The nginx config is huge: do we really need all of this?
Can we just default config with few modifications?

Reply:
The default really lacks a bit, but I can discard this, it's just a default scaffolding that I use on nginx images

Reply:
Yes, please discard it. Adds only required configurations.